plugin/postgres/plugin.go

```diff
@@ -141,6 +143,13 @@ func (p PostgresPlugin) Validate(endpoint ShieldEndpoint) error {
 		ansi.Printf("@G{\u2713 pg_password}  @C{%s}\n", s)
 	}
 
+	s, err = endpoint.StringValue("pg_database")
+	if err != nil {
+		ansi.Printf("@R{\u2717 pg_database  %s}\n", err)
```

Comment:
Additionally, this code needs to take into account the "all databases" vs "this database" distinction, and print out a message like:

```
pg_database         none (all databases will be backed up)
```

to let the operator know that all databases will be backed up.